2025-04-16 09:31:13 [ERROR]: שגיאה בהתחברות לוואטסאפ: WAConnection is not a constructor
TypeError: WAConnection is not a constructor
    at WhatsAppService.initialize (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\services\whatsappService.js:20:19)
    at Object.<anonymous> (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\app.js:41:17)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:170:5)
    at node:internal/main/run_main_module:36:49
2025-04-16 09:31:43 [ERROR]: שגיאה בהתחברות למסד הנתונים:
MongooseServerSelectionError
    at Connection.openUri (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\connection.js:847:32)
    at C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\index.js:351:10
    at C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\helpers\promiseOrCallback.js:31:10)
    at Mongoose._promiseOrCallback (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\index.js:1149:10)
    at Mongoose.connect (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\index.js:350:20)
    at Object.connect (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\database\database.js:16:22)
    at Object.<anonymous> (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\app.js:27:10)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:170:5)
2025-04-16 09:34:31 [ERROR]: שגיאה בהתחברות למסד הנתונים:
MongooseServerSelectionError
    at Connection.openUri (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\connection.js:847:32)
    at C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\index.js:351:10
    at C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\helpers\promiseOrCallback.js:31:10)
    at Mongoose._promiseOrCallback (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\index.js:1149:10)
    at Mongoose.connect (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\node_modules\mongoose\lib\index.js:350:20)
    at Object.connect (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\database\database.js:16:22)
    at Object.<anonymous> (C:\Users\Danny.LAPTOP-BQF1GMNR.000\Documents\GitHub\whatsapp-bot\whatsapp-event-bot\app.js:27:10)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:170:5)
2025-04-16 11:01:07 [ERROR]: לא התקבל קוד QR אחרי אתחול מחדש
2025-04-16 11:01:18 [ERROR]: לא התקבל קוד QR אחרי אתחול מחדש
2025-04-16 11:04:11 [ERROR]: לא התקבל קוד QR אחרי אתחול מחדש
2025-04-16 11:04:21 [ERROR]: לא התקבל קוד QR אחרי אתחול מחדש
2025-04-16 11:07:50 [ERROR]: לא ניתן לקבל קוד QR תקף
2025-04-16 11:08:05 [ERROR]: לא ניתן לקבל קוד QR תקף
2025-04-16 11:10:19 [ERROR]: לא ניתן לקבל קוד QR תקף
2025-04-16 11:10:35 [ERROR]: לא ניתן לקבל קוד QR תקף
2025-04-16 11:10:50 [ERROR]: לא ניתן לקבל קוד QR תקף
2025-04-16 22:14:43 [ERROR]: לא ניתן לקבל קוד QR תקף
2025-04-16 22:14:58 [ERROR]: לא ניתן לקבל קוד QR תקף
